meta-mentor-staging/recipes-core/udev/udev-extraconf/mount_modified.sh

```diff
@@ -19,10 +19,17 @@ done
 
 automount() {	
 	name="`basename "$DEVNAME"`"
-        if [ -z ${ID_FS_LABEL} ]; then
-		#Lets use UUID as the LABEL, as it will be unique for the $DEVNAME
-		ID_FS_LABEL=`/sbin/blkid | grep $DEVNAME | awk {'print $2'} | tr '=|"' ' ' | awk {'print $2'}`
```

Comment:
@Arsalan-Awan, as discussed earlier I believe this technique is not accurate as well so please identify that in the commit message.